src/I18n/RelativeTimeFormatter.php

```diff
@@ -115,7 +115,6 @@ public function timeAgoInWords(DateTimeInterface $time, array $options = [])
         }
 
         if ($diff > abs($now - (new FrozenTime($options['end']))->format('U'))) {
-            /** @var \Cake\I18n\DateFormatTrait $time */
```

Comment:
We shouldnt remove them, but then assign them, otherwise the method is not followable or clickable.
Also IDE cannot detect for us if the argument count or type is OK here.

Reply:
This inline docComment doesn't exist on master either, it was added on this branch itself to try to make phpstan happy.

Reply:
I see.
Maybe it was added somewhere else then in a similar way.
Well, we could still keep it here too :)